src/run

```diff
@@ -19,8 +19,8 @@ base_dir=$(cd "`dirname "$loc"`" && pwd)
 # Determine os version and choose correspondent .sh or .bat file accordingly.
 os_version="`uname -a`"
 
-if [[ $os_version == *"MINGW"* ]]; then
-  exec "$base_dir/target/start.bat" "$@"
-else
+if [[ $os_version == *"MINGW"* ]]; then  
+  cmd //C "$base_dir/target/start.bat" "$@"
```

Comment:
This change confuses me... if you're removing the start.bat stuff but then running it here.

And you switched back to cmd instead of exec.  I tried to make it just do no .bat stuff originally but something like sbt or something was doing it internally and I couldn't make it not.